pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -3,10 +3,16 @@ package release_payload_controller
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/release-controller/pkg/apis/release/v1alpha1"
 	releasepayloadclient "github.com/openshift/release-controller/pkg/client/clientset/versioned/typed/release/v1alpha1"
 	releasepayloadinformer "github.com/openshift/release-controller/pkg/client/informers/externalversions/release/v1alpha1"
 	releasepayloadlister "github.com/openshift/release-controller/pkg/client/listers/release/v1alpha1"
+	"github.com/openshift/release-controller/pkg/releasepayload/conditions"
+	"github.com/openshift/release-controller/pkg/releasepayload/jobstatus"
+	releasepayloadhelpers "github.com/openshift/release-controller/pkg/releasepayload/v1alpha1helpers"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"reflect"
```

Comment:
missing the writes and reads doc on the controller.

Reply:
Done!